portability-types-transfer/src/main/java/org/datatransferproject/types/transfer/retry/RetryingCallable.java

```diff
@@ -85,6 +92,11 @@ public T call() throws RetryException {
           if (nextAttemptIntervalMillis > 0L) {
             try {
               Thread.sleep(nextAttemptIntervalMillis);
+              monitor.debug(
+                  () ->
```

Comment:
Does it make sense to move this log line up to the beginning of the try block? (where the attempt actually happens)? Also what do you think of this format? "Attempt %d started, service:%s, dataType: %s" to be consistent with the log above.

Reply:
If we move this log line to the beginning, then we would make the logging even for the first try. And first try is not a retry, so that's why I have put it here.
Having said that, no strong feeling either way. I can make that change if you think that is better.

Reply:
I think thats fine, the log would just look like "Attempt 1 started..." - just a thought since it would avoid using that `attempt + 1` in the format string